index.html

```diff
@@ -1412,6 +1412,104 @@ <h3>Biometrics</h3>
     </section>
   </section>
 
+  <section class="informative">
+    <h2>Environmental and Ethical Considerations</h2>
+
+    <p>
+      The following section details certain key areas of focus when implementing a 
+      new DID method, or implementing a solution that incorporates DIDs.
+    </p>
+
+    <p>
+      The <a 
+      href="https://www.w3.org/2001/tag/doc/ethical-web-principles/">W3C TAG
+      Ethical Web Principles</a> document is a helpful guide
+      when weighing ethical considerations of various technologies.
+    </p>
+
+    <section class="informative">
+      <h3>Broad Ethical Principles</h3>
+      <p>
+        Decentralized Identifiers may come to underpin much of our digital life. This 
+        may include public social and career personas, as well as more private personas 
+        shared among friends and family. Identifiers representing these personas and 
+        relationships may be some of the most important in our lives, so great care 
+        will be taken when choosing an identifier system to support these critical 
+        activities. As with all things, the appropriate and ethical use of technology
+        will be strongly considered  when implementing items related to DIDs.
+      </p>
+
+      <p>
+        As noted in the <a
+        href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#principles">
+        Principles</a> section of the Ethical Web Principles, there are key goals that
+        apply to all Web standards and technologies. DIDs are explicitly intended to
+        help users get closer to several of these goals, especially the following:
+        <ul>
+          <li>
+            <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#privacy">
+              Privacy
+            </a>
+          </li>
+          <li>
+            <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#control">
+              Individual Control
+            </a>
+          </li>
+          <li>
+            <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#multi">
+              Device Independence
+            </a>
+          </li>
+        </ul>
+      </p>
+    </section>
+
+    <section class="informative">
+      <h3>Environmental Considerations</h3>
+      <p class="advisement">
+        The following section reflects the views of some members of the working group.
+        Additional PRs are welcome from the working group with additional points of view.
+      </p>
+
+      <section class="informative">
+        <h3>Environmental Principles as Applied to DID method development</h3>
```

Comment:
per @ChristopherA's request
```suggestion
        <h3> Other concerns as applied to DID method development</h3>
```